app/redux/constants/errorMessages.js

```diff
@@ -1,11 +1,19 @@
 import i18next from '../../core/language';
+import config from '../../config';
 
 const t = i18next.t.bind(i18next);
 
 // relating to signup
 export const ERR_ACCOUNT_ALREADY_EXISTS = t('That e-mail address already has an account.');
 // relating to login
 export const ERR_LOGIN_CREDS = t('Wrong username or password.');
+export const errLoginLocked = () => {
+  const params = {
+    delayBeforeNextLoginAttempt: config.DELAY_BEFORE_NEXT_LOGIN_ATTEMPT,
+    maxFailedLogin: config.MAX_FAILED_LOGIN_ATTEMPTS,
+  };
+  return t('Wrong username or password. Your account may have been lock for {{delayBeforeNextLoginAttempt}} minutes after {{maxFailedLogin}} failed attempts.', params);
+}
```

Comment:
Something that we have to review with Marine
```suggestion
  return t('Wrong username or password. For your security your account has been locked for {{delayBeforeNextLoginAttempt}} minutes because you have reached the maximum number of logon attempts.', params);
}
```